daycare/test_daycare.py

```diff
@@ -37,72 +25,39 @@ def test_daycare_user_delete(self, create_daycare_user):
         create_daycare_user.user.delete()
         assert create_daycare_user not in DayCare.objects.all()
 
-    def test_daycare_user_creation_with_invalid_email(self):
-        with pytest.raises(ValidationError, match="'Enter a valid email address.'"):
-            DayCare.create(username='testuser02', email="invalid mail format", password=PASSWORD,
-                           name=NAME, description=DESCRIPTION,
-                           price_per_day=PRICE_PER_DAY,
-                           capacity=CAPACITY, area=AREA, city=CITY, address=ADDRESS)
+    @pytest.mark.parametrize(
+        ['email', 'name', 'price_per_day', 'capacity',
+         'area', 'city', 'address', 'expected_error_message'],
+        [
+            pytest.param('invalid mail format', 'test_name', 10, 50, 'Merkaz', 'tel-aviv', 'test address',
+                         "'Enter a valid email address.'"),
+            pytest.param('test@gmail.com', 'this is very long name above 20 letters', 10, 50, 'Merkaz', 'tel-aviv',
+                         'test address', 'Invalid name - max length is 20'),
+            pytest.param('test@gmail.com', 'test_name', 'invalid price', 50, 'Merkaz', 'tel-aviv', 'test address',
+                         'Invalid price_per_day - price should be number.'),
+            pytest.param('test@gmail.com', 'test_name', 10000, 50, 'Merkaz', 'tel-aviv', 'test address',
+                         'Invalid price - max price is 4 digits.'),
+            pytest.param('test@gmail.com', 'test_name', 10, 50, 'this is very long name above 20 letters', 'tel-aviv',
+                         'test address', 'Invalid area - max length is 20'),
+            pytest.param('test@gmail.com', 'test_name', 10, 50, 'Merkaz', 'this is very long name above 20 letters',
+                         'test address', 'Invalid city - max length is 20'),
+            pytest.param('test@gmail.com', 'test_name', 10, 50, 'Merkaz', 'tel-aviv',
+                         'this is very long name above 50 letters and maybe moreeeeeee',
+                         'Invalid address - max length is 50')
+        ],
```

Comment:
I'd suggest adding an `id="test description"` field to each `pytest.param`. you don't have to include it in the params list.

Reply:
done